unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MinimapMetadata.cs

```diff
@@ -84,24 +89,45 @@ public class MinimapSceneInfo
         public string description;
         public string previewImageUrl;
 
-        [NonSerialized]
-        private int hashCode = -1;
+        [NonSerialized] private int? cachedHash;
+
+        public bool Equals(MinimapSceneInfo other)
+        {
+            if (ReferenceEquals(null, other)) return false;
+            if (ReferenceEquals(this, other)) return true;
+
+            // skip `previewImageUrl` on purpose
+            return name == other.name
+                   && type == other.type
+                   && parcels.SequenceEqual(other.parcels)
+                   && isPOI == other.isPOI
+                   && owner == other.owner
+                   && description == other.description
+                   && previewImageUrl == other.previewImageUrl;
```

Comment:
You mention in the comment that you are skipping the previewImageUrl, but its still being compared. Is this intentional?

Reply:
no, thanks